recipes/stages/classification/supcon.yaml

```diff
@@ -0,0 +1,21 @@
+_base_: [
+    './train.yaml',
+    '../_base_/data/twocrop_data.py',
+    '../_base_/models/cls_supcon.py',
+]
+
+runner:
+    max_epochs: 20
+
+optimizer_config:
+    type: SAMOptimizerHook
+
+optimizer:
+    type: SGD
+    lr: 0.005
+    momentum: 0.9
+    weight_decay: 0.0005
+
+evaluation:
+    metric: ['accuracy', 'class_accuracy']
+
```

Comment:
Could you explain why task adapt op disappear in this recipe?f

Reply:
Task adapt replaces whichever loss function I specify in the model by CrossEntropyLoss.

It wasn't working before, but maybe after the refactoring I pushed this morning, it might work.

I will check and revert it if it works.

Reply:
Yes, I've run a couple of tests, and it works now with `task_adapt` after the refactoring. Thank you!